### PR TITLE
Add Eslint and update packages to meet linting requirements 

### LINKED ISSUE #17: Add ESLint Config
This repository should have ESLint Config.

```
.eslintrc.js
.prettierrc
```
We should use the examples here, for reference:
https://github.com/BBC-News/simorgh/blob/latest/.eslintrc.js
https://github.com/BBC-News/simorgh/blob/latest/.prettierrc

Dependencies required include: 
```
babel-eslint
eslint
eslint-config-airbnb
eslint-config-prettier
eslint-plugin-import
eslint-plugin-json
eslint-plugin-jsx-a11y
eslint-plugin-prettier
prettier
```
## Tasks
- Set up the eslint and prettier config files, with correct dependencies.
- Add a lint script to the `package.json` file `test:lint` that runs the eslint command.
- When `npm run test:lint` is run, we should get all the lint errors from the files 
- This PR should also fix any lint errors in the repo.

### PR DESCRIPTION
**Pull request by dr3**
Resolves #17 

Adds eslint rules from Simorgh, installs dependancies and fixes linting

 - [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "storybook": "NODE_ENV=development start-storybook -p 8080 -c .storybook",
     "storybook:export": "build-storybook -c .storybook -o public",
     "speculate": "speculate",
-    "test": "jest --verbose"
+    "test": "jest --verbose",
```
Comment (ChrisBAshton) [inline package.json]: Shall we copy [Simorgh's approach](https://github.com/BBC-News/simorgh/blob/latest/package.json#L23) and have this run `npm run test:lint && npm run test:unit`?

File: package.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
...
+    "test:lint": "eslint --ext .js,jsx,json ./packages"
```
Comment (ChrisBAshton) [inline package.json]: Looks good for now. May need to add additional directories RE discussion in #19.

**Review by ChrisBAshton — DISMISSED** [review]
LGTM 👍 most comments can be addressed organically in future PRs

**Review by bcmn — DISMISSED** [review]

**Review by sareh — CHANGES_REQUESTED** [review]
Looks great. 

However, when I run this locally, I get these lint errors:

```

psammead/packages/example-button/src/index.story.jsx
  4:39  error  Unable to resolve path to module 'styled-components'  import/no-unresolved

psammead/packages/example-button/src/styles/styledExampleButton.js
  1:20  error  Unable to resolve path to module 'styled-components'  import/no-unresolved
```

It'd be good to fix this error in this PR.

**Review by sareh — APPROVED** [review]
👍 Great. Linting works well. `npm run test:lint` & `npm run test`
We can have the unit tests fixed in the separate issue #16

**Review by bcmn — APPROVED** [review]

**Pull request merged**